make_queue/templates/make_queue/course/registration_edit.html

```diff
@@ -25,6 +25,15 @@ <h1>
         </h1>
 
         {% block messages %}
+            <div id="messages" style="margin-bottom:20px; display: none;">
+                <div class="ui red message">
+                    <i class="close icon"></i>
```

Comment:
I think the outer div is not needed. If you remove it, the `message` class already has a margin on the bottom. And you can use `ui hidden red message` as the class, to hide the message initially. Then just us an id on the `ui red message` div to easily select it with jQuery.

Might also be better to use `warning` instead of `red`, as this is not necessarily an error (Dataporten is not limited to NTNU users only, however, LDAP is).